src/Ombi/ClientApp/src/app/media-details/components/shared/media-poster/media-poster.component.html

```
@@ -1,10 +1 @@
-<div class="col-md-2 col-sm-3 hidden-xs">
-    <div class="sidebar sidebar-poster affixable affix-top">
-        <div class="poster mobile-poster">
-            <ombi-image class="real" [src]="posterPath" alt="Poster"
-                style="display: block;"></ombi-image>
-        </div>
-        <!--Underneith poster-->
-        <br />
-    </div>
-</div>
\ No newline at end of file
+<ombi-image class="poster-img" [src]="posterPath" alt="Poster"></ombi-image>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)